### PR TITLE
Tabindex edits

### PR DESCRIPTION
**Pull request by apastan**
### Description

1. [Update the warning section and recommended values for tabindex](https://github.com/mdn/content/commit/dd03dcf09fd6dd97c081fbedcb86e9f989035c03)

Developers should use caution when using values other than 0 or −1 for their [tabindex](https://html.spec.whatwg.org/multipage/interaction.html#attr-tabindex) attributes as this is complicated to do correctly.

2. [Remove duplicate content and move all information about usage into Note](https://github.com/mdn/content/commit/e522ab0394ba6e99028d55ddedf9813b6516b299)

3. [Delete the duplicate and contradictory content](https://github.com/mdn/content/commit/6c990fed4a9e625cda68fd46758454c2713ccb1e)

3.1 Duplicate content:

The `tabindex` attribute must not be specified on `dialog` elements and **should not be used on non-interactive content.** 

and than

**Avoid using the `tabindex` attribute in conjunction with non-[interactive content](/en-US/docs/Web/Guide/HTML/Content_categories#interactive_content)** to make something intended to be interactive focusable by keyboard input.

3.2 Contradictory content

**If you set the `tabindex` attribute on a `div`**, then its child content cannot be scrolled with the arrow keys unless you set `tabindex` on the content, too. [Check out this fiddle to understand the scrolling effects of `tabindex`](https://jsfiddle.net/jainakshay/0b2q4Lgv/).

and than

**Avoid using the `tabindex` attribute in conjunction with non-[interactive content](/en-US/docs/Web/Guide/HTML/Content_categories#interactive_content)** to make something intended to be interactive focusable by keyboard input.

4. [Add info about tabindex default value](https://github.com/mdn/content/commit/0d198cb7eba2887ceef4ebe1d942d5dafa7afc1b)

The tabIndex IDL attribute must [reflect](https://html.spec.whatwg.org/multipage/common-dom-interfaces.html#reflect) the value of the [tabindex](https://html.spec.whatwg.org/multipage/interaction.html#attr-tabindex) content [a](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-a-element)ttribute. The default value is 0 if the element is an a, [area](https://html.spec.whatwg.org/multipage/image-maps.html#the-area-element), [button](https://html.spec.whatwg.org/multipage/form-elements.html#the-button-element), [frame](https://html.spec.whatwg.org/multipage/obsolete.html#frame), [iframe](https://html.spec.whatwg.org/multipage/iframe-embed-object.html#the-iframe-element), [input](https://html.spec.whatwg.org/multipage/input.html#the-input-element), [object](https://html.spec.whatwg.org/multipage/iframe-embed-object.html#the-object-element), [select](https://html.spec.whatwg.org/multipage/form-elements.html#the-select-element), [textarea](https://html.spec.whatwg.org/multipage/form-elements.html#the-textarea-element), or [SVG a](https://svgwg.org/svg2-draft/linking.html#AElement) element, or is a [summary](https://html.spec.whatwg.org/multipage/interactive-elements.html#the-summary-element) element that is a [summary for its parent details](https://html.spec.whatwg.org/multipage/interactive-elements.html#summary-for-its-parent-details). The default value is −1 otherwise.

[Source](https://html.spec.whatwg.org/multipage/interaction.html#dom-tabindex)

5. [Correct mistake](https://github.com/mdn/content/commit/921ad67ff735fcfd3fbdc327966bdf89fe176b57)

<blockquote>If not specified, it takes the default value 0.</blockquote>

omitted (or non-integer values)
The user agent will decide whether the element is [focusable](https://html.spec.whatwg.org/multipage/interaction.html#focusable), and if it is, whether it is [sequentially focusable](https://html.spec.whatwg.org/multipage/interaction.html#sequentially-focusable) or [click focusable](https://html.spec.whatwg.org/multipage/interaction.html#click-focusable) (or both).

[Source](https://html.spec.whatwg.org/multipage/interaction.html#the-tabindex-attribute)

6. [Add info about tabindex value equals 0](https://github.com/mdn/content/commit/b572452dbd4a451bc35d43a71890deb6bb7c351a)

``tabindex="0" causes the element to be [focusable](https://html.spec.whatwg.org/multipage/interaction.html#focusable), and indicates that the author would prefer the element to be both [click focusable](https://html.spec.whatwg.org/multipage/interaction.html#click-focusable) and [sequentially focusable](https://html.spec.whatwg.org/multipage/interaction.html#sequentially-focusable). The user agent might ignore this preference for click and sequential focusability.

[Source](https://html.spec.whatwg.org/multipage/interaction.html#the-tabindex-attribute)

7.  [Fix link with 301 redirect](https://github.com/mdn/content/pull/22882/commits/acba0b07a2434bef7a8d0879b5dce7bd4c3c5b25)

8. [Update tabindex definition](https://github.com/mdn/content/pull/22882/commits/89bd60ca899658bf322127fd9100391f8e9cd425)

[Source](https://html.spec.whatwg.org/multipage/interaction.html#attr-tabindex)

### CODE REVIEW

**Comment by apastan** [comment]
If it's possible I would like to add a new article about `focusable` elements on MDN, because I didn't find any information about this. May be someone could create empty file to allow me edit it?

File: files/en-us/web/html/global_attributes/tabindex/index.md [hunk 1]
```diff
@@ -10,26 +10,27 @@ browser-compat: html.global_attributes.tabindex
...
+- A _negative value_ (the exact negative value doesn't actually matter, usually `tabindex="-1"`) means that the element is not reachable via sequential keyboard navigation, but could be focused with JavaScript (by calling the [`focus()`](/en-US/docs/Web/API/HTMLElement/focus) method) or visually by clicking with the mouse.
 
-  > **Note:** A negative value is useful when you have off-screen content that appears on a specific event. The user won't be able to focus any element with a negative `tabindex` using the keyboard, but a script can do so by calling the `focus()` [method](/en-US/docs/Web/API/HTMLElement/focus).
+  > **Note:** `tabindex="-1"` may be useful for elements that should not be navigated to directly using the <kbd>Tab</kbd> key, but need to have keyboard focus set to them. Examples include a modal dialog window that should be focused when it is opened, or a form submission error message that should be immediately focused when an errant form is submitted.
```
Comment (estelle) [inline files/en-us/web/html/global_attributes/tabindex/index.md:21]: drive by comment: tabindex should never be set on a dialog. https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog#attributes

Comment (apastan) [reply files/en-us/web/html/global_attributes/tabindex/index.md:21]: Your are absolutly right. The tabindex attribute must not be used on the <dialog> element. I will add it.
https://html.spec.whatwg.org/multipage/interactive-elements.html#the-dialog-element

Here I mean the case when we have a hidden (off-screen) dialog window for instance, login window or, (not exactly html-element <dialog>) which is shouldn't be reachable with keyboard (Tab key) but there are <input> elements in this dialog window which should be focused with JavaScript on some event (on button click or form submission).

`modal dialog window` sounds confusing. I can offer:

```suggestion
  > **Note:** `tabindex="-1"` may be useful for elements that should not be navigated to directly using the <kbd>Tab</kbd> key, but need to have keyboard focus set to them. Examples include a off-screen modal window that should be focused when it is opened, or a form submission error message that should be immediately focused when an errant form is submitted.
```

File: files/en-us/web/html/global_attributes/tabindex/index.md [hunk 2]
```diff
@@ -10,26 +10,29 @@ browser-compat: html.global_attributes.tabindex
...
+- If the `tabindex` value is omitted the user agent will decide whether the element is focusable.
 
-  > **Warning:** Avoid using `tabindex` values greater than 0. Doing so makes it difficult for people who rely on assistive technology to navigate and operate page content. Instead, write the document with the elements in a logical sequence.
+  > **Warning:** It is recommended that developers use only 0 and -1 as `tabindex` values. Avoid using `tabindex` values greater than 0 and css-properties that can change the order of focusable html-elements ([Ordering flex items](/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Ordering_Flex_Items)). Doing so makes it difficult for people who rely on using keyboard for navigation or assistive technology to navigate and operate page content. Instead, write the document with the elements in a logical sequence.
```
Comment (Josh-Cena) [inline files/en-us/web/html/global_attributes/tabindex/index.md:27]:
```suggestion
  > **Warning:** You are recommended to only use `0` and `-1` as `tabindex` values. Avoid using `tabindex` values greater than `0` and CSS properties that can change the order of focusable HTML elements ([Ordering flex items](/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Ordering_Flex_Items)). Doing so makes it difficult for people who rely on using keyboard for navigation or assistive technology to navigate and operate page content. Instead, write the document with the elements in a logical sequence.
```

**Review by Josh-Cena — COMMENTED** [review]
Purely editorial suggestions

File: files/en-us/web/html/global_attributes/tabindex/index.md [hunk 3]
```diff
@@ -10,26 +10,29 @@ browser-compat: html.global_attributes.tabindex
 
 {{HTMLSidebar("Global_attributes")}}
 
-The **`tabindex`** [global attribute](/en-US/docs/Web/HTML/Global_attributes) indicates that its element can be focused, and where it participates in sequential keyboard navigation (usually with the <kbd>Tab</kbd> key, hence the name).
+The **`tabindex`** [global attribute](/en-US/docs/Web/HTML/Global_attributes) allows developers to make html-elements focusable, allow or prevent them from being sequentially focusable (usually with the <kbd>Tab</kbd> key, hence the name), and determine their relative ordering for sequential focus navigation.
```
[the remark this answers is not in the document]
Comment (Josh-Cena) [reply files/en-us/web/html/global_attributes/tabindex/index.md:13]: I find "make HTML elements focusable, allow or prevent them from being sequentially focusable" slightly repetitive. There's probably some overlapping semantics here but I don't have good wordings to suggest, and I'm fine to only make the capitalization change.

File: files/en-us/web/html/global_attributes/tabindex/index.md [hunk 4]
```diff
@@ -10,26 +10,29 @@ browser-compat: html.global_attributes.tabindex
...
 
-- `tabindex="0"` means that the element should be focusable in sequential keyboard navigation, after any positive tabindex values and its order is defined by the document's source order.
-- A _positive value_ means the element should be focusable in sequential keyboard navigation, with its order defined by the value of the number. That is, `tabindex="4"` is focused before `tabindex="5"` and `tabindex="0"`, but after `tabindex="3"`. If multiple elements share the same positive `tabindex` value, their order relative to each other follows their position in the document source. The maximum value for `tabindex` is 32767. If not specified, it takes the default value 0.
+- `tabindex="0"` means that the element should be focusable in sequential keyboard navigation, after any positive tabindex values and its order is defined by the document's source order. The element could be focused with JavaScript (by calling the [`focus()`](/en-US/docs/Web/API/HTMLElement/focus) method) or visually by clicking with the mouse.
```
[the remark this answers is not in the document]
Comment (apastan) [reply files/en-us/web/html/global_attributes/tabindex/index.md:23]: You are right, it is duplicated from above. You got my point. `0` and `-1` are the most used as `tabindex` values and I was going to emphasizes that elements with `tabindex=”0”` also have programmatic focus and can be focused by users with mouse.

You are right, `tabindex` doesn’t fully cover topic `Focusable elements in HTML` because there are more cases when an element can become focusable (browsers have a list of elements which are focusable by default even without specified tablindex attribute). Tabindex value can be overridden by CSS (if an element or its parents have display: none or visibility:hidden) or `hidden` attribute and these elements will not be focusable and tabindex value really doesn't matter. For example, element with `display: none` and `tabindex=5` will not be focusable. And may be all these is out `tabindex` if scope.

But `tabindex` is relevant to the topic ` focusable elements in HTML` because developer can make an element focusable by adding ` tabindex` attribute with valid value.

To summarize, if an HTML element has `tabindex` attribute with any of valid value (for example, -3, -1, 0, 1, 5) this element can be focused with JavaScript (by developers) or visually by clicking by users with the mouse (or tapping on mobile) for every `tabindex` value.

And agree with your offer and I can remove this point from each item in the list and will explicitly specify this point before or after the list. Let me know if I need to do this.

File: files/en-us/web/html/global_attributes/tabindex/index.md [hunk 5]
```diff
@@ -10,26 +10,29 @@ browser-compat: html.global_attributes.tabindex
...
 
-The `tabindex` attribute must not be specified on {{htmlelement("dialog")}} elements and should not be used on non-interactive content. If you set the `tabindex` attribute on a {{htmlelement("div")}}, then its child content cannot be scrolled with the arrow keys unless you set `tabindex` on the content, too. [Check out this fiddle to understand the scrolling effects of `tabindex`](https://jsfiddle.net/jainakshay/0b2q4Lgv/).
+Some focusable html-elements are determined (by [user agent](/en-US/docs/Glossary/User_agent) under hood) to have `tabindex` default value equals 0. These elements are an {{HTMLElement("a")}} and {{HTMLElement("area")}} with `href` attribute, {{HTMLElement("button")}}, {{HTMLElement("frame")}} {{deprecated_inline}}, {{HTMLElement("iframe")}}, {{HTMLElement("input")}}, {{HTMLElement("object")}}, {{HTMLElement("select")}}, {{HTMLElement("textarea")}}, or SVG {{SVGElement("a")}} element, or is a {{HTMLElement("summary")}} element that is a summary for its parent {{HTMLElement("details")}}. Developers shouldn't add the `tabindex` attribute to these elements only if they are going to change the default behavior.
```
[the remark this answers is not in the document]
Comment (Josh-Cena) [reply files/en-us/web/html/global_attributes/tabindex/index.md:29]:
```suggestion
Some focusable HTML elements have a default `tabindex` value of 0 set under the hood by the [user agent](/en-US/docs/Glossary/User_agent). These elements are an {{HTMLElement("a")}} or {{HTMLElement("area")}} with `href` attribute, {{HTMLElement("button")}}, {{HTMLElement("frame")}} {{deprecated_inline}}, {{HTMLElement("iframe")}}, {{HTMLElement("input")}}, {{HTMLElement("object")}}, {{HTMLElement("select")}}, {{HTMLElement("textarea")}}, or SVG {{SVGElement("a")}} element, or a {{HTMLElement("summary")}} element that provides summary for a {{HTMLElement("details")}} element. Developers shouldn't add the `tabindex` attribute to these elements unless it changes the default behavior.
```

Makes sense, but it should be "under the hood" in that case 😄

[the remark this answers is not in the document]
Comment (apastan) [reply files/en-us/web/html/global_attributes/tabindex/index.md:29]: Oops, non-native speaker is detected 😄. 

Sorry, I have some problems with translation. Thanks for your quick answers and fixes.

File: files/en-us/web/html/global_attributes/tabindex/index.md [hunk 6]
```diff
@@ -10,26 +10,31 @@ browser-compat: html.global_attributes.tabindex
...
+The **`tabindex`** [global attribute](/en-US/docs/Web/HTML/Global_attributes) allows developers to make HTML elements focusable, allow or prevent them from being sequentially focusable (usually with the <kbd>Tab</kbd> key, hence the name) and determine their relative ordering for sequential focus navigation.
 
 {{EmbedInteractiveExample("pages/tabbed/attribute-tabindex.html","tabbed-standard")}}
 
 It accepts an integer as a value, with different results depending on the integer's value:
 
-- A _negative value_ (usually `tabindex="-1"`) means that the element is not reachable via sequential keyboard navigation, but could be focused with JavaScript or visually by clicking with the mouse. It's mostly useful to create accessible widgets with JavaScript.
+  > **Note:** If an HTML element renders and has `tabindex` attribute with any valid integer's value, so this element can be focused with JavaScript (by calling the [`focus()`](/en-US/docs/Web/API/HTMLElement/focus) method) or visually by clicking with the mouse.
```
Comment (estelle) [inline files/en-us/web/html/global_attributes/tabindex/index.md:19]:
```suggestion
  > **Note:** If an HTML element renders and has `tabindex` attribute with any valid integer value, the element can be focused with JavaScript (by calling the [`focus()`](/en-US/docs/Web/API/HTMLElement/focus) method) or visually by clicking with the mouse. If the value is `0` or greater, the element can also be focused via the tab key.
```

File: files/en-us/web/html/global_attributes/tabindex/index.md [hunk 7]
```diff
@@ -10,26 +10,31 @@ browser-compat: html.global_attributes.tabindex
...
 
-- `tabindex="0"` means that the element should be focusable in sequential keyboard navigation, after any positive tabindex values and its order is defined by the document's source order.
-- A _positive value_ means the element should be focusable in sequential keyboard navigation, with its order defined by the value of the number. That is, `tabindex="4"` is focused before `tabindex="5"` and `tabindex="0"`, but after `tabindex="3"`. If multiple elements share the same positive `tabindex` value, their order relative to each other follows their position in the document source. The maximum value for `tabindex` is 32767. If not specified, it takes the default value 0.
+  > **Note:** `tabindex="-1"` may be useful for elements that should not be navigated to directly using the <kbd>Tab</kbd> key, but need to have keyboard focus set to them. Examples include a off-screen modal window that should be focused when it is opened, or a form submission error message that should be immediately focused when an errant form is submitted.
```
Comment (estelle) [inline files/en-us/web/html/global_attributes/tabindex/index.md:23]:
```suggestion
  > **Note:** `tabindex="-1"` may be useful for elements that should not be navigated to directly using the <kbd>Tab</kbd> key, but need to have keyboard focus set to them. Examples include an off-screen modal window that should be focused when it comes into view, or a form submission error message that should be immediately focused when an errant form is submitted.
```

File: files/en-us/web/html/global_attributes/tabindex/index.md [hunk 8]
```diff
@@ -10,26 +10,31 @@ browser-compat: html.global_attributes.tabindex
...
 
-  > **Warning:** Avoid using `tabindex` values greater than 0. Doing so makes it difficult for people who rely on assistive technology to navigate and operate page content. Instead, write the document with the elements in a logical sequence.
+- `tabindex="0"` means that the element should be focusable in sequential keyboard navigation, after any positive tabindex values. Focus navigation order of these elements is defined by their order in the document's source.
```
Comment (estelle) [inline files/en-us/web/html/global_attributes/tabindex/index.md:25]:
```suggestion
- `tabindex="0"` means that the element should be focusable in sequential keyboard navigation, after any positive tabindex values. Focus navigation order of these elements is defined by their order in the document source.
```

File: files/en-us/web/html/global_attributes/tabindex/index.md [hunk 9]
```diff
@@ -10,26 +10,31 @@ browser-compat: html.global_attributes.tabindex
...
+  > **Note:** `tabindex="-1"` may be useful for elements that should not be navigated to directly using the <kbd>Tab</kbd> key, but need to have keyboard focus set to them. Examples include a off-screen modal window that should be focused when it is opened, or a form submission error message that should be immediately focused when an errant form is submitted.
 
-  > **Warning:** Avoid using `tabindex` values greater than 0. Doing so makes it difficult for people who rely on assistive technology to navigate and operate page content. Instead, write the document with the elements in a logical sequence.
+- `tabindex="0"` means that the element should be focusable in sequential keyboard navigation, after any positive tabindex values. Focus navigation order of these elements is defined by their order in the document source.
```
Comment (estelle) [inline files/en-us/web/html/global_attributes/tabindex/index.md:25]:
```suggestion
- `tabindex="0"` means that the element should be focusable in sequential keyboard navigation, after any positive `tabindex` values. The focus navigation order of these elements is defined by their order in the document source.
```

File: files/en-us/web/html/global_attributes/tabindex/index.md [hunk 10]
```diff
@@ -10,26 +10,31 @@ browser-compat: html.global_attributes.tabindex
...
+- A _positive value_ means the element should be focusable in sequential keyboard navigation, with its order defined by the value of the number. That is, `tabindex="4"` is focused before `tabindex="5"` and `tabindex="0"`, but after `tabindex="3"`. If multiple elements share the same positive `tabindex` value, their order relative to each other follows their position in the document source. The maximum value for `tabindex` is 32767.
+- If the `tabindex` value is omitted the user agent will decide whether the element is focusable.
```
Comment (estelle) [inline files/en-us/web/html/global_attributes/tabindex/index.md:27]:
```suggestion
- If the `tabindex` attribute is included with no value set, whether the element is focusable is determined by the user-agent.
```

File: files/en-us/web/html/global_attributes/tabindex/index.md [hunk 11]
```diff
@@ -10,26 +10,31 @@ browser-compat: html.global_attributes.tabindex
...
+  > **Warning:** You are recommended to only use `0` and `-1` as `tabindex` values. Avoid using `tabindex` values greater than `0` and CSS properties that can change the order of focusable HTML elements ([Ordering flex items](/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Ordering_Flex_Items)). Doing so makes it difficult for people who rely on using keyboard for navigation or assistive technology to navigate and operate page content. Instead, write the document with the elements in a logical sequence.
+
+Some focusable HTML elements have a default `tabindex` value of 0 set under the hood by the [user agent](/en-US/docs/Glossary/User_agent). These elements are an {{HTMLElement("a")}} or {{HTMLElement("area")}} with `href` attribute, {{HTMLElement("button")}}, {{HTMLElement("frame")}} {{deprecated_inline}}, {{HTMLElement("iframe")}}, {{HTMLElement("input")}}, {{HTMLElement("object")}}, {{HTMLElement("select")}}, {{HTMLElement("textarea")}}, or SVG {{SVGElement("a")}} element, or a {{HTMLElement("summary")}} element that provides summary for a {{HTMLElement("details")}} element. Developers shouldn't add the `tabindex` attribute to these elements unless it changes the default behavior.
```
Comment (estelle) [inline files/en-us/web/html/global_attributes/tabindex/index.md:31]:
```suggestion
Some focusable HTML elements have a default `tabindex` value of `0` set under the hood by the [user agent](/en-US/docs/Glossary/User_agent). These elements are an {{HTMLElement("a")}} or {{HTMLElement("area")}} with `href` attribute, {{HTMLElement("button")}}, {{HTMLElement("frame")}} {{deprecated_inline}}, {{HTMLElement("iframe")}}, {{HTMLElement("input")}}, {{HTMLElement("object")}}, {{HTMLElement("select")}}, {{HTMLElement("textarea")}}, or SVG {{SVGElement("a")}} element, or a {{HTMLElement("summary")}} element that provides summary for a {{HTMLElement("details")}} element. Developers shouldn't add the `tabindex` attribute to these elements unless it changes the default behavior.
```

**Review by estelle — COMMENTED** [review]
A few suggested edits

File: files/en-us/web/html/global_attributes/tabindex/index.md (hunk 10 above)
Comment (apastan) [reply files/en-us/web/html/global_attributes/tabindex/index.md:27]: user-agent => user agent
```suggestion
- If the `tabindex` attribute is included with no value set, whether the element is focusable is determined by the user agent.
```

File: files/en-us/web/html/global_attributes/tabindex/index.md (hunk 11 above)
Comment (apastan) [reply files/en-us/web/html/global_attributes/tabindex/index.md:31]:
```suggestion
Some focusable HTML elements have a default `tabindex` value of `0` set under the hood by the [user agent](/en-US/docs/Glossary/User_agent). These elements are an {{HTMLElement("a")}} or {{HTMLElement("area")}} with `href` attribute, {{HTMLElement("button")}}, {{HTMLElement("frame")}} {{deprecated_inline}}, {{HTMLElement("iframe")}}, {{HTMLElement("input")}}, {{HTMLElement("object")}}, {{HTMLElement("select")}}, {{HTMLElement("textarea")}}, and SVG {{SVGElement("a")}} element, or a {{HTMLElement("summary")}} element that provides summary for a {{HTMLElement("details")}} element. Developers shouldn't add the `tabindex` attribute to these elements unless it changes the default behavior (for instance, including a negative value will remove the element from the focus navigation order).
```

File: files/en-us/web/html/global_attributes/tabindex/index.md (hunk 6 above)
Comment (apastan) [reply files/en-us/web/html/global_attributes/tabindex/index.md:19]: We discussed this point with @Josh-Cena - https://github.com/mdn/content/pull/22882#discussion_r1047734948. 

The idea was to specifiy common behavior of HTML-elements with `tabindex` attribute before the list to avoid duplicates in each item of this list.

Please, consider the following:

```suggestion
> **Note:** If an HTML element renders and has `tabindex` attribute with any valid integer value, the element can be focused with JavaScript (by calling the [`focus()`](/en-US/docs/Web/API/HTMLElement/focus) method) or visually by clicking with the mouse. The particular `tabindex` value controls whether the element is `tabbable` (i.e. reachable via sequential keyboard navigation, usually with the <kbd>Tab</kbd> key).
```

File: files/en-us/web/html/global_attributes/tabindex/index.md [hunk 12]
```diff
@@ -10,26 +10,31 @@ browser-compat: html.global_attributes.tabindex
...
+
+Some focusable HTML elements have a default `tabindex` value of `0` set under the hood by the [user agent](/en-US/docs/Glossary/User_agent). These elements are an {{HTMLElement("a")}} or {{HTMLElement("area")}} with `href` attribute, {{HTMLElement("button")}}, {{HTMLElement("frame")}} {{deprecated_inline}}, {{HTMLElement("iframe")}}, {{HTMLElement("input")}}, {{HTMLElement("object")}}, {{HTMLElement("select")}}, {{HTMLElement("textarea")}}, and SVG {{SVGElement("a")}} element, or a {{HTMLElement("summary")}} element that provides summary for a {{HTMLElement("details")}} element. Developers shouldn't add the `tabindex` attribute to these elements unless it changes the default behavior (for instance, including a negative value will remove the element from the focus navigation order).
```
Comment (estelle) [inline files/en-us/web/html/global_attributes/tabindex/index.md:31]:
```suggestion
Some focusable HTML elements have a default `tabindex` value of `0` set under the hood by the [user agent](/en-US/docs/Glossary/User_agent). These elements are an {{HTMLElement("a")}} or {{HTMLElement("area")}} with `href` attribute, {{HTMLElement("button")}}, {{HTMLElement("frame")}} {{deprecated_inline}}, {{HTMLElement("iframe")}}, {{HTMLElement("input")}}, {{HTMLElement("object")}}, {{HTMLElement("select")}}, {{HTMLElement("textarea")}}, and SVG {{SVGElement("a")}} element, or a {{HTMLElement("summary")}} element that provides summary for a {{HTMLElement("details")}} element. Developers shouldn't add the `tabindex` attribute to these elements unless it changes the default behavior (for example, including a negative value will remove the element from the focus navigation order).
```

**Review by estelle — APPROVED** [review]
